src/main/java/org/terasology/potions/effect/ExplosiveEffect.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2016 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.potions.effect;
+
+import org.terasology.entitySystem.entity.EntityManager;
+import org.terasology.entitySystem.entity.EntityRef;
+import org.terasology.entitySystem.event.ReceiveEvent;
+import org.terasology.logic.actions.ActionTarget;
+import org.terasology.logic.actions.ExplosionActionComponent;
+import org.terasology.logic.health.DoDamageEvent;
+import org.terasology.potions.HerbEffect;
+import org.terasology.potions.events.DrinkPotionEvent;
+import org.terasology.world.block.Block;
+
+/**
+ * This effect destroys all blocks in a specified block radius

```

Comment:
It doesn't actually destroy *all* blocks - that seems more up to how the explosion is configured, and our "standard" explosion is pretty lax about just how much it actually destroys :-)